src/three-components/SmoothControls.ts

```diff
@@ -449,7 +481,8 @@ export class SmoothControls extends EventDispatcher {
         (MINIMUM_DAMPENING_FACTOR - MAXIMUM_DAMPENING_FACTOR)
   }
 
-  private[$userAdjustOrbit](deltaTheta: number, deltaPhi: number, deltaRadius: number): boolean {
+  private[$userAdjustOrbit](
+      deltaTheta: number, deltaPhi: number, deltaRadius: number): boolean {
```

Comment:
Thanks for running `clang-format`. We need to set this up to run on a pre-commit hook...

Filed https://github.com/GoogleWebComponents/model-viewer/issues/530

Reply:
I must have not had it set up at first, because a bunch of new formatting changes came in this time.